polynote-frontend/polynote/ui/component/notebook/notebookconfig.ts

```diff
@@ -118,7 +118,7 @@ export class NotebookConfigEl extends Disposable {
 
             try {
                 paste = JSON.parse(clipText);
-                let resolvers = Resolvers.parseWrappedResolvers(paste.repositories);
+                let resolvers = Resolvers.parseWrappedResolvers((paste.repositories as unknown) as [{type: "ivy" | "maven" | "pip", resolver: RepositoryConfig}]);
```

Comment:
Hm. I _ think_ you can do this a little more cleanly (though still kind of ugly) by modifying the type of paste. Rather than `NotebookConfig`, try something like: `Omit<NotebookConfig, "repositories"> & {repositories: [{type: "ivy" | "maven" | "pip", resolver: RepositoryConfig}]}`

https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys

Reply:
And a type alias for `{type: "ivy" | "maven" | "pip", resolver: RepositoryConfig}` might be a good idea